packages/graphql/src/scripts/2025-07-10_drop_live_session_cache_data.ts

```diff
@@ -0,0 +1,24 @@
+import { Redis } from 'ioredis'
+
+// ? This script removes all redis cache entries that are related to live sessions (and not the new live quiz)
+async function run() {
+  const redisExec = new Redis({
+    family: 4,
+    host: process.env.REDIS_HOST ?? 'localhost',
+    password: process.env.REDIS_PASS ?? '',
+    port: Number(process.env.REDIS_PORT) ?? 6379,
+    tls: process.env.REDIS_TLS ? {} : undefined,
+  })
+
+  // drop all live session redis cache entries starting with 's:'
+  const keys = await redisExec.keys('s:*')
+  console.log(`Deleting ${keys.length} keys:`)
+  console.log(keys)
+
+  // TODO: uncomment this line to execute redis pipeline
+  // await redisExec.del(...keys)
```

Comment:
```suggestion
  // await redisExec.unlink(...keys)
```

I think the unlink command is more efficient as it is not blocking (redis will just do it at some point with garbage cleaning)